app/models/status_message.rb

```diff
@@ -58,7 +58,7 @@ def self.tag_stream(tag_ids)
   end
 
   def nsfw
-    text.try(:match, /#nsfw/i) || super
+    !!(text.try(:match, /#nsfw/i) || super)
```

Comment:
Style/DoubleNegation: Avoid the use of double negation (`!!`).

Reply:
Aw, common